recipes/easytest/meta.yaml

```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "easytest" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "bfecf6663b14bd5418e2d36b615ccdf7" %}
```

Comment:
I'm unsure what "pipit" is or which documentation you are referring to, but you cannot use an md5 hash in the place of a sha256 hash. Please refer to https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L7-L10 or in the case of packages where the source is pypy.io, you can just click on the "sha256" link on the package page: 

https://pypi.org/project/easytest/#files